src/controllers/ajax/IndexAjaxController.php

```diff
@@ -157,10 +158,12 @@ protected function getActions(): array<string> {
       // This will help avoid leaking users ldap passwords if the server's database
       // is compromised.
       $ldap_password = $password;
-      $password = strval(gmp_strval(
-        gmp_init(bin2hex(openssl_random_pseudo_bytes(16)), 16),
-        62,
-      ));
+      $password = strval(
+        gmp_strval(
+          gmp_init(bin2hex(openssl_random_pseudo_bytes(16)), 16),
```

Comment:
while you're touching this, changing this to use `random_bytes(16)` would be good - and potentially using the raw bytes, hex, or base64 instead of requiring GMP to do a base62 conversion

Reply:
Good idea.  We can swap this out in a few places while we are at it.  Currently `openssl_random_pseudo_bytes()` is in use in:

* IndexAjaxController (twice).
* Integration
* Token.